File: comms/bulk-messaging/sender-pools/add-sender-to-pool/add-sender-to-pool.py
from twilio_comms import TwilioComms

client = TwilioComms(
    account_id="TWILIO_ACCOUNT_SID",
    auth_token="TWILIO_AUTH_TOKEN",
)

client.sender_pools.add_sender(
    sender_pool_id="comms_senderpool_01h9krwprkeee8fzqspvwy6nq8",
    request=[
        {
            "sender_id": "comms_sender_01h9krwprkeee8fzqspvwy6nq8"
        }
    ],
)

File: comms/bulk-messaging/sender-pools/add-sender-to-pool/add-sender-to-pool.php
<?php

namespace Example;

use Twilio\Comms\TwilioComms;
use Twilio\Comms\SenderPools\Types\SenderPoolsAddSenderRequestItem;

$client = new TwilioComms(
    accountId: 'TWILIO_ACCOUNT_SID',
    authToken: 'TWILIO_AUTH_TOKEN',
);
$client->senderPools->addSender(
    'comms_senderpool_01h9krwprkeee8fzqspvwy6nq8',
    [
        new SenderPoolsAddSenderRequestItem([
            'senderId' => 'comms_sender_01h9krwprkeee8fzqspvwy6nq8',
        ]),
    ],
);
